common.proto

```diff
@@ -22,17 +22,14 @@ package common;
 //
 // An absolute ID MUST be unique among all items in that set, but they do not
 // need to be unique globally.
-//
-// Note that absolute IDs do not match character-for-character between the
-// different backend implementations, but they MUST decode to the same
-// components.
 
 // When sent by a chat backend, the ID MAY optionally be used as a hint to the
 // frontend to get the same external ID on reconnection. When sent by core to a
 // plugin, the ID MUST be an ID pointing to an exact backend instance.
 message Backend {
   string id = 1;
   string type = 2;
+  string callback_url = 3;
```

Comment:
I think either url or uri makes sense - I'm pretty sure it'll stay one of those. And because this is proto, we can rename it without breaking the wire protocol later.